spec/controllers/alchemy/admin/pictures_controller_spec.rb

```diff
@@ -120,14 +120,7 @@ module Alchemy
       end
 
       context 'without passing validations' do
```

Comment:
Minor nit here: You changed this wording to "with failing validations", which I find more legible. Can you also do that here?